### DISCUSSION TITLE
How to reduce sync time from 10 mins to less.

### DISCUSSION THREAD

**Discussion by shivamsood** [argoproj/argo-cd#6066] [Q&A] [answered]
If my app is not able to deploy. I get the app degraded notification in around 10 mins. How can I reduce this time?

**Comment by shivamsood** [comment]
can someone please help here?

**Comment by alexmt** [comment, marked as answer]
I think you are looking for `.spec.progressDeadlineSeconds` https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#progress-deadline-seconds . Default value is 600 seconds. You can specify lower value in deployment spec.

**Reply by shivamsood** [reply]
Thank you so much @alexmt
